### PR TITLE
add install hint

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Signed-off-by: jetzlstorfer <juergen.etzlstorfer@dynatrace.com>

### CODE REVIEW

**Comment by jetzlstorfer** [comment]
Preview: 
image

File: site/tutorials/snippets/08/monitoring/setupPrometheus.md [hunk 1]
```diff
@@ -4,29 +4,31 @@ Duration: 3:00
...
+```
+
+<!-- 
+bash wait_for_deployment_in_namespace "prometheus-service" "keptn" 
+bash wait_for_deployment_in_namespace "prometheus-service-monitoring-configure-distributor" "keptn" 
+sleep 10
+-->
     
+Negative
+: Please note that Prometheus will be installed only after executing the next command. At this step, we have only installed the service that is responsible for managing Prometheus.
```
Comment (donovanmuller) [inline site/tutorials/snippets/08/monitoring/setupPrometheus.md:21]:
```suggestion
: Please note that a Keptn managed Prometheus will be installed only after executing the next command. At this step, we have only installed the service that is responsible for managing Prometheus.
```

**Comment by jetzlstorfer** [comment]
Preview URL: https://deploy-preview-125--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-08/index.html?index=..%2F..keptn08x#12

File: site/tutorials/snippets/08/monitoring/setupPrometheus.md (hunk 1 above)
Comment (donovanmuller) [inline site/tutorials/snippets/08/monitoring/setupPrometheus.md:21]:
```suggestion
: Please note you are not required to install Prometheus or have an existing Prometheus deployment on your cluster. A Keptn managed Prometheus will be installed only after executing the next command. At this step, we have only installed the service that is responsible for managing Prometheus.
```

**Review by donovanmuller — COMMENTED** [review]
Two suggestions to make it more explicit but happy with proposed as is as well.

File: site/tutorials/snippets/08/monitoring/setupPrometheus.md (hunk 1 above)
Comment (jetzlstorfer) [reply site/tutorials/snippets/08/monitoring/setupPrometheus.md]: Thank you!

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**